tests/misc/typescript-babel-only/tuple-labeled.ts

```diff
@@ -0,0 +1,22 @@
+// https://github.com/babel/babel/pull/11754
+// TODO: move this file to `test/typescript/tuple`
```

Comment:
??

Reply:
Current typescript-estree cannot parse labeled tuple types. So we should put this in `/tests/misc/typescript-babel-only` for now.